ads/microad.md

```diff
@@ -22,14 +22,14 @@ limitations under the License.
 <amp-ad width="320" height="50"
 	type="microad"
     data-spot="b4bf837c5ddd69758d5ac924d04cf502"
-    data-url=""
-    data-referrer="">
+    data-url="${COMPASS_EXT_URL}"
```

Comment:
Because these values are automatically generated, users are not required to set them.